.github/workflows/elementor-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Elementor Tests / Init
+
+on: [push, pull_request]
```

Comment:
why do you want on `push` and `pull_request` and not only `pull_request` event?

Reply:
In case it's a direct merge like after fix merge conflicts.